src/ssl.c

```diff
@@ -71,6 +79,10 @@ static struct {
         uint16_t port;
         int text_protocol;
     } memcached;
+    struct {
+        char *host;
+        uint16_t port;
+    } redis;
```

Comment:
Can we merge `conf.memcached` and `conf.redis` into a `union`, named `address` or something?